README.md

```diff
@@ -0,0 +1,148 @@
+  <h1>🍕 RepoQuery 🍕</h1>
+<p align="center">
+  <img src="https://img.shields.io/github/languages/code-size/open-sauced/repo-query" alt="GitHub code size in bytes">
+  <img src="https://img.shields.io/github/commit-activity/w/open-sauced/repo-query" alt="GitHub commit activity">
+  <a href="https://discord.gg/U2peSNf23P">
+    <img src="https://img.shields.io/discord/714698561081704529.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2" alt="Discord">
+  </a>
+  <a href="https://twitter.com/saucedopen">
+    <img src="https://img.shields.io/twitter/follow/saucedopen?label=Follow&style=social" alt="Twitter">
+  </a>
+</p>
+
+<img alt="Open Sauced" src="https://user-images.githubusercontent.com/46051506/255266242-0b94b779-3020-455e-8a26-d3ed8e292214.gif" width="600px">
+
+
+<p><h3>A REST service to answer user-queries about public GitHub repositories</h3></p>
+
+</div>
+
+## 🔎 The Project
+RepoQuery is an early-beta project, that uses recursive [OpenAI function calling](https://platform.openai.com/docs/api-reference/chat/create#chat/create-functions) paired with semantic search using [All-MiniLM-L6-V2](https://huggingface.co/rawsh/multi-qa-MiniLM-distill-onnx-L6-cos-v1/blob/main/onnx/model_quantized.onnx) to index and answer user queries about public GitHub repositories.
+
+##  📬 Service Endpoints
+
+> **Note:**
+Since the service returns responses as SSEs, a REST client like Postman is recommended. Download it [here](https://www.postman.com/downloads/). The Postman web client doesn't support requests to `localhost`.
+
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/18073744-276b793e-f5ec-418f-ba0a-9dff94af543e?action=collection%2Ffork&source=rip_markdown&collection-url=entityId%3D18073744-276b793e-f5ec-418f-ba0a-9dff94af543e%26entityType%3Dcollection%26workspaceId%3D8d8a1363-ad0a-45ad-b036-ef6a37e44ef8)
+
+### 1. `/embed`
+To generate and store embeddings for a GitHub repository.
+
+#### Parameters
+The parameters are passed as a JSON object in the request body:
+
+- `owner` (string, required): The owner of the repository.
+- `name` (string, required): The name of the repository.
+- `branch` (string, required): The name of the branch.
+
+#### Response
+The request is processed by the server and responses are sent as [Server-sent events(SSE)](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events). The event stream will contain the following events with optional data. https://github.com/open-sauced/repo-query/blob/f2f415a4fa9c02d4530624fd7bac2105eea1a77c/src/routes/events.rs#L14-L20
+
+#### Example
+```bash
+curl --location 'localhost:3000/embed' \
+--header 'Content-Type: application/json' \
+--data '{
+    "owner": "open-sauced",
+    "name": "ai",
+    "branch": "beta"
+}'
+```
+
+### 2. `/query`
+To perform a query on the API with a specific question related to a repository.
```

Comment:
Same here: this is a `POST`

Reply:
https://github.com/open-sauced/repo-query/pull/1/commits/3ec893a487f6c806e750ec5e8e398d81024d9fd6